Dockerfile

```diff
@@ -7,6 +7,9 @@ RUN npm install -g configurable-http-proxy
 RUN python3 -m ensurepip && \
     pip3 install --upgrade pip setuptools wheel pycurl
 
+RUN pip3 install pyvis graphviz pydotplus pytz intervaltree deprecation tqdm stringdist pyemd jsonpickle sympy numpy pandas 
```

Comment:
```suggestion
    pip3 install --upgrade pip setuptools wheel pycurl pyvis graphviz pydotplus pytz intervaltree deprecation tqdm stringdist pyemd jsonpickle sympy numpy pandas 
```

Reply:
Was testing dependencies so kept these separate for pm4py